lib/fika/cli.ex

```diff
@@ -45,4 +73,7 @@ defmodule Fika.Cli do
       {:error, _} -> :error
     end
   end
+
+  defp print_usage?,
+    do: :fika |> Application.fetch_env!(__MODULE__) |> Keyword.fetch!(:print_usage)
```

Comment:
Can this be replaced with `Application.get_env(:fika, :print_usage)`? Also, looks like we don't need the default in config/config.exs because that will be nil and that's falsey.

Reply:
No longer relevant, but there are some considerations:

- I prefer scoping things as much as I can, just so configs are easier to find.
- I chose to not use defaults so `fetch_env!` is used, because I think that makes things less error prone